### ISSUE TITLE
Update production, staging, and redesign about page

### ISSUE THREAD

**Issue by chigby** [freedomofpress/pressfreedomtracker.us#1273]
The about page is in an error state right now, caused by the `LogoListBlock` component.

The body field of the about page (and possibly other pages) contains obsolete data that needs to be migrated to a new format. The `LogoListBlock` has been superseded by the `InfoTableBlock` in the redesign. IMO it is not possible to automatically convert the logo list into an info table because the info table contains more information (namely: a piece of text that functions as the "title" of the link). So I think it's necessary to manually recreate the logo list as an info table for it to work.

This should be done before we remove the old fields.

**Comment by harrislapiroff** [comment]
This is now covered by #1333
